examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -767,6 +783,277 @@
     "`ConditionalDimension` can be used as an alternative to save snaps to disk as illustrated in [08_snapshotting](https://github.com/devitocodes/devito/blob/master/examples/seismic/tutorials/08_snapshotting.ipynb \"08_snapshotting\"). The example subsamples the `TimeDimension`, capturing snaps of the our wavefield at equally spaced snaps. "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use case C (Count nonzero elements/ Find nonzero positions)\n",
```

Comment:
More than a "use case", this is an example. 

Also, what the example is about should not end up within parentheses